pkg/ottl/contexts/ottlmetric/metrics.go

```diff
@@ -38,8 +42,20 @@ type TransformContext struct {
 	resourceMetrics      pmetric.ResourceMetrics
 }
 
+// MarshalLogObject serializes the metric into a zapcore.ObjectEncoder for logging.
+func (tCtx TransformContext) MarshalLogObject(encoder zapcore.ObjectEncoder) error {
```

Comment:
We don't offer any stability guarantees for these and therefore don't need a changelog here, right?

Reply:
Correct